src/publishers/github.js

```diff
@@ -31,7 +31,8 @@ export default async (artifacts, packageJSON, forgeConfig, authToken, tag) => {
           repo: forgeConfig.github_repository.name,
           tag_name: tag || `v${packageJSON.version}`,
           name: tag || `v${packageJSON.version}`,
-          draft: true,
+          draft: forgeConfig.github_repository.draft !== false,
```

Comment:
`--flags` is kind of weird because `draft` defaults to true.

> Mainly because I don't feel a change in publish target should require a commit into VC.

Hmm. How about this: `config.forge` are the "defaults" for the project, CLI can override certain fields, like `draft` and `prerelease`.